lib/matplotlib/cbook/__init__.py

```diff
@@ -678,33 +678,43 @@ def clear(self):
 
     def bubble(self, o):
         """
-        Raise *o* to the top of the stack.  *o* must be present in the stack.
+        Raise all references of *o* to the top of the stack, and return it.
 
-        *o* is returned.
+        Raises
+        ------
+        ValueError
+            If *o* is not in the stack.
         """
         if o not in self._elements:
-            raise ValueError('Unknown element o')
-        old = self._elements[:]
+            raise ValueError('Given element not contained in the stack.')
```

Comment:
exception messages (typically) have no final dot (this is consistent with the stdlib, and also with most of the rest of mpl)